src/components/ebay-3d-viewer/examples/02-with-options/template.marko

```diff
@@ -0,0 +1,16 @@
+class {}
+
+<ebay-3d-viewer
+    src='https://ir.ebaystatic.com/cr/v/c1/ebayui/3d/v1/image.glb'
```

Comment:
How come some are double-quoted while this one is single.

Reply:
It's because I am used to using prettier and it's not working locally. 😅